### PR TITLE
LPS-165995 | Test Automation | Master

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-165995

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
@JhossephG, great job on the test. This one only needs some minor changes. Thank you!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -1018,6 +1018,39 @@ definition {
...
+				fieldLabelName = "Custom Long Int Field",
+				fieldName = "customLongIntField",
+				fieldType = "Clob",
+				isRequired = "false",
+				objectName = "CustomObject165995");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject165995");
+		}
+
+		task ("When a entry more than 16 digits is aded") {
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1041]: Typo: you mean "added"

**Comment by evanilson-santana** [comment]
Hey, @JhossephG can you fix the Typo and forward to the Bchan?

**Comment by JhossephG** [comment]
ci:forward

**Pull request closed without merging**